hpat/tests/test_series.py

```diff
@@ -122,6 +122,55 @@ def gen_strlist(size, nchars=8):
     return [''.join(chars) for chars in generated_chars]
 
 
+def restore_series_argsort(series, my_result):
+    nona = series.dropna()
+    data = np.copy(nona.data)
+    new_result = np.copy(series.data)
+    result = np.copy(my_result)
+    for i in range(len(result)):
+        if result[i] != -1:
+            new_result[i] = data[result[i]]
+        else:
+            new_result[i] = np.nan
+    return new_result
+
+
+#if indices right - return 0; wrong - return 1
+def restore_series_sort_values(series, my_result_index, ascending):
```

Comment:
Please, add doc-string to clarify what this func does

Reply:
Fixed